test/test_automl.py

```diff
@@ -344,6 +345,32 @@ def test_micro_macro_f1(self):
         print(multi_class_curves(y_train, y_pred_proba, roc_curve))
         print(multi_class_curves(y_train, y_pred_proba, precision_recall_curve))
 
+    def test_roc_auc(self):
+        automl_experiment_binary = AutoML()
+        automl_settings = {
+            "time_budget": 2,
+            "metric": "roc_auc",
+            "task": "classification",
+            "log_file_name": "test/roc_auc_binary.log",
+            "log_training_metric": True,
+            "n_jobs": 1,
+            "model_history": True
+        }
+        X_train, y_train = load_breast_cancer(return_X_y=True)
+        automl_experiment_binary.fit(
+            X_train=X_train, y_train=y_train, **automl_settings)
+
+        automl_experiment_multi = AutoML()
+        automl_settings["multi_class"] = "ovr"
```

Comment:
I think there will be an error when you pass `multi_class="ovr"` to the `fit()` of the underlying estimators, such as LGBMClassifier.
To avoid that error, an alternative way I can suggest is to use 'roc_auc_ovr' and 'roc_auc_ovo' as the new metric name and remove the `multi_class` argument from `fit_kwargs`.